<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amazon SimpleDB GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/main/icon.ico</normaloff>:/main/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horlayCreate">
     <item>
      <widget class="QLabel" name="labelDB">
       <property name="text">
        <string>Database:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineDbName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushCreateDB">
       <property name="text">
        <string>Create!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushRemoveDB">
       <property name="text">
        <string>Remove!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushListDB">
       <property name="text">
        <string>List!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushMetaDB">
       <property name="text">
        <string>Info!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horlayItem">
     <item>
      <widget class="QLabel" name="labelItemName">
       <property name="text">
        <string>Item:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineItem"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushItemPut">
       <property name="text">
        <string>Item put</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushItemGet">
       <property name="text">
        <string>Item get</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horlayItemEdit">
     <item>
      <widget class="QPushButton" name="pushAttrAdd">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushAttrRemove">
       <property name="text">
        <string>Remove...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushAttrClear">
       <property name="text">
        <string>Clear...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableItemAttributes">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainAnswer">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horlayOkConfig">
     <item>
      <widget class="QPushButton" name="pushConfig">
       <property name="text">
        <string>Config</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineDbName</tabstop>
  <tabstop>pushCreateDB</tabstop>
  <tabstop>pushRemoveDB</tabstop>
  <tabstop>pushListDB</tabstop>
  <tabstop>pushMetaDB</tabstop>
  <tabstop>lineItem</tabstop>
  <tabstop>pushItemPut</tabstop>
  <tabstop>pushItemGet</tabstop>
  <tabstop>pushAttrAdd</tabstop>
  <tabstop>pushAttrRemove</tabstop>
  <tabstop>pushAttrClear</tabstop>
  <tabstop>tableItemAttributes</tabstop>
  <tabstop>pushConfig</tabstop>
  <tabstop>pushOK</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushOK</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAttrClear</sender>
   <signal>clicked()</signal>
   <receiver>tableItemAttributes</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
